### ISSUE TITLE
Add the aliases '..' and '...'

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#3738]
The aliases `..` and `...` for going up the directory tree are fairly standard. But they are not available in the Docker container and sometimes I trip over that.

```
otobo@ba4f7f35a106:~/Kernel$ ..
bash: ..: command not found
```

Let's add it to _.bash_aliases_ .

**Comment by bschmalhofer** [comment]
Merged in #3741, sorry misspelled the issue number again and said #3838 in the commit message.

A quick test looked fine. Closing this issue.

**Comment by bschmalhofer** [comment]
Having the aliases `..`and `...` are fine for all release series. So let's add them.
